iot-mqtt-common/src/test/resources/devicePrivateKey.cer

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
**DO NOT** embedded **real device** credential info,  even for testing !!! Watch out security incident alert.
P.S. unit testing doesn't need to make any real requests to our service.

Reply:
Removed in next commit.